app/src/components/MatricDivider.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import { Box, ResponsiveHidingView, ScreenSize } from '@kibalabs/ui-react';
+
+export const MatricDivider = (): React.ReactElement => {
+  return (
+    <ResponsiveHidingView hiddenBelow={ScreenSize.Medium}>
+      <Box variant='divider' height='70px' width='1px' />
```

Comment:
this height should not be hardcoded. lets find a better way to do this (probably full height with some padding)